ydb/core/blobstorage/nodewarden/node_warden_pdisk.cpp

```diff
@@ -278,6 +282,24 @@ namespace NKikimr::NStorage {
             return;
         }
 
+        const auto [_, inserted] = PDiskRestartInFlight.emplace(pdiskId);
+
+        if (!inserted) {
+            // Restart is already in progress.
+            return;
+        }
+
+        const TPDiskKey pdiskKey(LocalNodeId, pdiskId);
+
+        const TVSlotId from(pdiskKey.NodeId, pdiskKey.PDiskId, 0);
+        const TVSlotId to(pdiskKey.NodeId, pdiskKey.PDiskId, Max<ui32>());
+
+        for (auto it = LocalVDisks.lower_bound(from); it != LocalVDisks.end() && it->first <= to; ++it) {
```

Comment:
should we really do it explicitly here? I thought it was expected that VDisks terminate in their own way when they find out that round in incorrect